src/metpy/plots/plot_areas.py

```diff
@@ -0,0 +1,1014 @@
+# Copyright (c) 2016,2017,2019 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Collection of pre-defined areas for plotting with associated projections."""
+
+from collections import namedtuple
+
+from .cartopy_utils import import_cartopy
+from ..package_tools import Exporter
+
+ccrs = import_cartopy()
+exporter = Exporter(globals())
+Areas = namedtuple('Area', ['area', 'name', 'bounds', 'projection'])
+
+area_data = {
+    'us': Areas('us', 'UNITED STATES',
+                (-119.0, -56.0, 19.0, 47.0),
+                ccrs.NorthPolarStereo(central_longitude=-100)),
+    'awips': Areas('awips', 'AWIPS US',
+                   (-127.0, -59.0, 20.0, 50.0),
+                   ccrs.LambertConformal(standard_parallels=[30, 60], central_longitude=-100)),
```

Comment:
Instead of a dict here, how about just a list for the data to reduce redundancy, and maybe shorten a bunch of these lines...(see below):
```suggestion
area_data = [
    Areas('us', 'UNITED STATES',
          (-119.0, -56.0, 19.0, 47.0),
          ccrs.NorthPolarStereo(central_longitude=-100)),
    Areas('awips', 'AWIPS US',
          (-127.0, -59.0, 20.0, 50.0),
          ccrs.LambertConformal(standard_parallels=[30, 60], central_longitude=-100)),
```